package.json

```diff
@@ -113,6 +115,7 @@
     "prop-types": "^15.7.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-native": "^0.71.7",
```

Comment:
I think we should match the Jobber mobile version instead of this version

Reply:
Good idea!